NGitLab/Models/LabelEdit.cs

```diff
@@ -11,14 +12,19 @@ public LabelEdit()
 
         public LabelEdit(int projectId, Label label)
```

Comment:
It would probably be nicer if we had a proper distinction whether it's projectId or groupId.

Could we have like:
- ProjectLabelEdit that takes a projectId
- GroupLabelEdit that takes a groupId
- Both inherits LabelEdit
- All methods that takes a LabelEdit will then been enforced with the correct type

Reply:
After discussion:
- Avoid making a breaking change
- But create new class with new methods along
- Do not put things as obsolete, but use EditorBrowsableAttribute for "soft" obsolete